scripts/asc_apply_submission_settings.sh

```
@@ -0,0 +1,227 @@
+        fi
+    fi
+    if [ "$rd_id" = "null" ]; then
+        if [ $DRY_RUN -eq 1 ]; then
+            mark_failed "$platform の審査連絡先が未作成"
+        else
+            mark_failed "$platform の審査連絡先を作成できなかった"
+        fi
+        continue
+    fi
+    CUR=$(get_review "$rd_id")
+    [ "$CUR" = "$WANT_REVIEW" ] && log "  [OK] $platform 審査連絡先" || mark_failed "$platform の審査連絡先が定義と不一致: $(mask_phone <<<"$CUR")"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)